hw/market.py

```diff
@@ -1,28 +1,59 @@
+from datetime import datetime
+
+
+def started_decorator(function):
+    def wrapper(*args, **kwargs):
+        start = datetime.now()
+        print(f'Function {function.__name__} started at: ' + str(start))
+        wrap_result = function(*args, **kwargs)
+        finish = datetime.now()
+        print('Finished at: ' + str(finish) + '. Execution time: ' + str(finish-start))
+        return wrap_result
+    return wrapper
+
+
 class Market:
     def __init__(self, wines: list = None, beers: list = None) -> None:
-        pass
+        drinks = wines + beers
+        self.drinks = {}
+        for drink in drinks:
+            self.drinks[drink.title] = drink
```

Comment:
в питоне такое можно сделать проще
```suggestion
        self.drinks = {drink.title: drink for drink in (wines + beers)}
```